jsm.go

```diff
@@ -727,6 +735,74 @@ func (js *js) StreamInfo(stream string, opts ...JSOpt) (*StreamInfo, error) {
 	return resp.StreamInfo, nil
 }
 
+func (js *js) StreamContainedSubjects(stream string, opts ...JSOpt) (map[string]uint64, error) {
+	if err := checkStreamName(stream); err != nil {
+		return nil, err
+	}
+	o, cancel, err := getJSContextOpts(js.opts, opts...)
+	if err != nil {
+		return nil, err
+	}
+	if cancel != nil {
+		defer cancel()
+	}
+
+	var i int
+	var subjectMessagesMap map[string]uint64 = nil
+
+	if o.streamInfoOpts != nil && o.streamInfoOpts.SubjectsFilter == _EMPTY_ {
+		o.streamInfoOpts.SubjectsFilter = ">"
+	}
+
+	for {
+		var req []byte
+
+		o.streamInfoOpts.Offset = i
```

Comment:
What if the user did not provide a filter, then o.streamInfoOpts will be nil and this will panic. You need to have a stack variable declared about (around line 753) and maybe set the SubjectsFilter based on the user provided one (if any). Something like this:
```
sio := streamInfoOpts{}
if o.streamInfoOpts != nil && ... {
   sio.SubjectFilter = ">"
}
for {
   ...
   sio.Offset = i
   if req, err = json.Marshal(&sio); err != nil {
        return nil, err
   }
...
```
Hope that makes sense.

Reply:
Actually I meant to override sio.SubjectFilter to anything that is passed by the user, so likely initialize sio with ">" and then override.
```
sio := streamInfoOpts{SubjectsFilter: ">"}
if o.streamInfoOpts != nil && o.streamInfoOpts.SubjectsFilter != _EMPTY_ {
  sio.SubjectsFilter = o.streamInfoOpts.SubjectsFilter
}
```
This is assuming that this API does not need any of the possible streamInfoOpts except for filter. Otherwise adapt the above.

Reply:
Yes, sorry I missed that, it's now replaced by the new version of StreamInfo in this last commit